stable/sysdig/templates/securitycontextconstraint.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and .Values.openshift.createSCC (.Capabilities.APIVersions.Has "security.openshift.io/v1") }}
```

Comment:
Don't rely on APIVersions.Has and let this step manual.

Please, also order as:

version
kind
metadata 

and the rest of the fields